frontend/src/utils/attribution-html.ts

```diff
@@ -163,15 +181,20 @@ export interface AttributionOptions {
 export const getAttribution = (
   mediaItem: AttributableMedia,
   i18n: VueI18n | null = null,
-  { includeIcons, isPlaintext }: AttributionOptions = {
+  { includeIcons, isPlaintext, isXml }: AttributionOptions = {
     isPlaintext: false,
     includeIcons: true,
+    isXml: false,
   }
 ): string => {
   if (!mediaItem) {
     return ""
   }
 
+  if (isXml) {
+    isPlaintext = true
+  }
```

Comment:
This might just be me, but this conflation of XML with plain text causes a bit of mental gymnastics in the logic below. I feel like using `isXml` and `isPlaintext` separately would've been clearer.

Reply:
Can we bypass the plaintext handling if we just use `mediaItem.attribution`, too, by the way?

Reply:
That could work, and it would cut down on the code the frontend duplicates from the API, but I'm not sure about localisation though. The frontend is capable of localising the attribution text in a way the API is not.

Reply:
Ah true :confused: In that case, should we undo the change from this comment before merging? https://github.com/WordPress/ openverse/pull/4499#discussion_r1643556075

Reply:
I don't a simple revert will be adequate. I'm personally not well versed with the i18n situation of the attribution box and feel like looking into what currently happens (and I expect should happen) will need a bit of time. Part of that's because I'm not entirely sure if XML, which is more geared towards machines than people, should even be translated.

I was only referring to the rich and plain text versions, which should be translated imo and using `mediaItem.attribution` is definitely not enough for those two types of attribution.

Reply:
@zackkrida suggested using just the license URL, and the DC standard recommendation matches that too:

> Typically, rights information includes a statement about various property rights associated with the resource, including intellectual property rights. Recommended practice is to refer to a rights statement with a URI. If this is not possible or feasible, a literal value (name, label, or short text) may be provided.

Then we don't need to worry about i18n at all for this (ideal!).

Reply:
@madewithkode would you be able to update this to use the license URL instead of the attribution text?

Reply:
@dhruvkb this would mean just switching `<dc:rights>${mediaItem.attribution}</dc:rights>` in the XML attribution snippet to reference the CC license URL as in `<dc:rights>${mediaItem.license_url}</dc:rights>` right?

Reply:
This has been updated.